source/AI.cpp

```diff
@@ -3356,7 +3356,8 @@ void AI::AimTurrets(const Ship &ship, FireCommand &command, bool opportunistic)
 			{
 				// Get the index of this weapon.
 				int index = &hardpoint - &ship.Weapons().front();
-				double offset = (hardpoint.HarmonizedAngle() - hardpoint.GetAngle()).Degrees();
+				Angle targetAngle = hardpoint.GetIdleAngle();
+				double offset = (targetAngle - hardpoint.GetAngle()).Degrees();
```

Comment:
targetAngle is unnecessary
```suggestion
				double offset = (hardpoint.GetIdleAngle() - hardpoint.GetAngle()).Degrees();
```